lib/evaluators/AsyncEvaluator.ts

```diff
@@ -35,6 +41,7 @@ export class AsyncEvaluator {
       exists: context.exists,
       aggregate: context.aggregate,
       bnode: context.bnode,
+      defaultTimeZone,
```

Comment:
Let's inline to simplify things.

```suggestion
      defaultTimeZone: context.defaultTimeZone || extractTimeZone(now),
```

(also in the other file)